site/en/r2/tutorials/generative/dcgan.ipynb

```diff
@@ -1,143 +1,197 @@
 {
```

Comment:
<p>Prefer imperative to "lets":</p><p><br></p><p>"Use the (as yet ..."</p><p><br></p><p>More generally, try to avoid "we" and "lets": https://developers.google.com/style/person</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Cool, made some updates.</p>